### PR TITLE
Docker stuff basically 

### PR DESCRIPTION
**Pull request by flamableassassin**

### CODE REVIEW

**Comment by flamableassassin** [comment]
Kinda messed up those comit msgs

File: bot.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ const eris = require('eris', {
     // - MESSAGE_DELETE_BULK
 });
 
+config.dev_mode = !(process.env.PROD || !config.dev_mode)
```
Comment (LiteLT) [inline bot.js:22]: It's more common to name the environment variable `NODE_ENV`, which could be equal to `development` or `production`. However, I don't care if you change this or not.

File: README.md [hunk 2]
```diff
@@ -7,11 +7,16 @@ You require:
...
 
-Steps to setup the bot
+Steps to setup the bot (NodeJS)
 1. Simply rename `example.config.js` to `config.js` and change the details to your liking.
 2. After that, simply do `npm install` to install all the dependencies.
 3. Finally, run the bot via `npm start`
 
+Steps to setup the bot (Docker)
+1. Simply rename `example.config.js` to `config.js` and change the details to your liking.
+2. Build the Docker image `Docker build -t chip:1.1.1 .`
```
Comment (LiteLT) [inline README.md:17]: Is the command supposed to be capital?

**Review by LiteLT — COMMENTED** [review]
I've never used Docker, so Adam and Like will have to take over.

File: bot.js (hunk 1 above)
Comment (flamableassassin) [reply bot.js:22]: I'm gonna leave it cause I don't really wanna have to rebuild it

Comment (A-da-m) [reply bot.js:22]: Use NODE_ENV, PROD makes no sense

File: README.md (hunk 2 above)
Comment (A-da-m) [reply README.md:17]: @LiteLT It's not supposed to be, I don't think it matters, but it does bug me.

**Review by A-da-m — APPROVED** [review]
To recap, switch PROD to NODE_ENV and rename the "Docker" command to start with "docker". Other than that, looks good to merge.

**Pull request merged**